LICENSE.txt

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Ben Balter
+Copyright (c) 2015-2017 Ben Balter and contributors
```

Comment:
Might as well wait 2 weeks and we can make this 2018. :smile:

Reply:
Ready to be merged on new year's eve 😄